AstSemantics.md

```diff
@@ -266,9 +266,11 @@ a value and may appear as children of other expressions.
 ### Branches and nesting
 
 The `br` and `br_if` constructs express low-level branching.
-Branches may only reference labels defined by an outer *enclosing construct*.
-This means that, for example, references to a `block`'s label can only occur 
-within the `block`'s body.
+Branches may only reference labels defined by an outer *enclosing construct*,
+which can be a `block` (with a label at the end), `loop` (with a label at the
+beginning), or the function body (with a label at the end). This means that,
```

Comment:
'or the function body (with a label at the end)' - this seems a significant new change too? Could this be deferred to discussion in a separate issue. One concern is that this might require an interpretation of the break arguments as the return values when the multi-value support has not been determined yet - this interpretation might be ok but there are different ways this could go.